onnxruntime/test/perftest/TFModelInfo.cc

```diff
@@ -48,7 +48,12 @@ std::unique_ptr<TestModelInfo> TFModelInfo::Create(_In_ const PATH_CHAR_TYPE* mo
     }
   }
 
+// To avoid breaking CentOS build
+#if defined(__GNUC__) && (__GNUC__ == 4)
   return std::move(ret);
```

Comment:
>std::move(ret) [](start = 9, length = 14)

If you do not do std::move() on return then it would not break anything. CentOS is correct. You should not apply std::move() on return values.